composer.json

```diff
@@ -69,6 +69,7 @@
         "prestashop/graphnvd3": "^2",
         "prestashop/gridhtml": "^2",
         "prestashop/gsitemap": "^4",
+        "prestashop/hummingbird": "dev-develop",
```

Comment:
```suggestion
        "prestashop/hummingbird": "~0.2",
```

Reply:
@jolelievre for the classic theme, on develop branch, we put `dev-develop`. We only put tags on the version branches.

Reply:
You're right it makes sense!